### PR TITLE
add support for g3 d852

### PR DESCRIPTION
**Pull request by anarcat**

### CODE REVIEW

**Comment by Lekensteyn** [comment]
Is it possible to add lsusb info?

    sudo lsusb -d 1004:631d -v > info/lsusb-d852.txt

You could optionally mask the last part of iSerial with `xxxx` if you want (see the existing info/lsusb-d855.txt file). This lsusb file could help later to determine what kind of changes are needed.

**Comment by anarcat** [comment]
sure - would love to see this phone fixed...

File: info/lsusb-d852.txt [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  bDeviceClass          239 Miscellaneous Device
+  bDeviceSubClass         2 ?
+  bDeviceProtocol         1 Interface Association
+  bMaxPacketSize0        64
+  idVendor           0x1004 LG Electronics, Inc.
+  idProduct          0x633a 
+  bcdDevice            2.32
+  iManufacturer           1 LG Electronics Inc.
+  iProduct                2 LGE Android Phone
+  iSerial                 3 LGD85XXXXXXXXX
```
Comment (Lekensteyn) [inline info/lsusb-d852.txt]: was this LGD852?

Comment (anarcat) [reply info/lsusb-d852.txt]: rah. yes, i probably stripped one too many... let me double-check that.

Comment (anarcat) [reply info/lsusb-d852.txt]: fixed.

File: rules.d/42-usb-lglaf.rules [hunk 2]
```diff
@@ -5,3 +5,5 @@ SUBSYSTEM=="usb", ATTRS{idVendor}=="1004", ATTRS{idProduct}=="633e", TAG+="uacce
 SUBSYSTEM=="usb", ATTRS{idVendor}=="1004", ATTRS{idProduct}=="627f", TAG+="uaccess"
 # LG G4 (VS986) in download mode
 SUBSYSTEM=="usb", ATTRS{idVendor}=="1004", ATTRS{idProduct}=="6298", TAG+="uaccess"
+# LG G3 (D852) in download mode
+SUBSYSTEM=="usb", ATTRS{idVendor}=="1004", ATTRS{idProduct}=="631d", TAG+="uaccess"
```
Comment (Lekensteyn) [inline rules.d/42-usb-lglaf.rules]: this ID is different from the lsusb above, is that intentional?

Comment (anarcat) [reply rules.d/42-usb-lglaf.rules]: not sure what happened there, quite frankly.  that's probably a mistake...

Comment (Lekensteyn) [reply rules.d/42-usb-lglaf.rules]: maybe it was in normal mode, can you double-check that this is the correct mode? Then I can merge it :)

Comment (anarcat) [reply rules.d/42-usb-lglaf.rules]: normal mode shows product 6300, iirc, and download mode is 633a, i rerolled the patch below.

**Comment by Lekensteyn** [comment]
Thanks, I hope you have some luck with unraveling secrets!

**Pull request merged**